src/components/ScenarioParameters/components/SaveLaunchDiscardButton.js

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Grid, Button, Typography } from '@mui/material';
+import { PermissionsGate } from '@cosmotech/ui';
+import SaveIcon from '@mui/icons-material/Save';
+import CancelIcon from '@mui/icons-material/Cancel';
+import PlayCircleOutlineIcon from '@mui/icons-material/PlayCircleOutline';
+import { useTranslation } from 'react-i18next';
+import { ACL_PERMISSIONS } from '../../../services/config/accessControl';
+import { useUserAppAndCurrentScenarioPermissions } from '../../../hooks/SecurityHooks';
+
+const SaveLaunchDiscardButton = ({ save, launch, saveAndLauch, discard, isDirty, runDisabled }) => {
```

Comment:
question about naming: usually we begin function's name in prop by _on_ (`onSave`, `onLaunch`, `onDiscard` etc) and boolean props begin with _is_ or _has_, isn't it relevant here as well ?